sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -573,10 +573,27 @@ private static String getConfigTemplateDownloadPath(ConfigFileSpec config) {
       containerInfo.getLinuxInfoBuilder().setSharePidNamespace(podSpec.getSharePidNamespace());
       // Isolate the tmp directory of tasks
       // switch to SANDBOX SELF after dc/os 1.13
+
       containerInfo.addVolumes(Protos.Volume.newBuilder()
           .setContainerPath("/tmp")
           .setHostPath("tmp")
           .setMode(Protos.Volume.Mode.RW));
+
+      LOGGER.info("Setting seccomp info unconfined: {} profile: {}",
+              podSpec.getSeccompUnconfined(),
+              podSpec.getSeccompProfileName());
+
+      containerInfo.getLinuxInfoBuilder().setSeccomp(Protos.SeccompInfo.newBuilder()
```

Comment:
I'd prefer to explicitly encode the XOR behaviour as mentioned under the second bullet under 'Notes' [here](https://jira.mesosphere.com/browse/DCOS-49133?focusedCommentId=254417&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-254417).

For the case where `podSpec.getSeccompProfileName().isPresent()` we can explicitly set 
`Protos.SeccompInfo.newBuilder().setUnconfined(false).build()` to be unambiguous an in full compliance with [this](https://github.com/apache/mesos/blob/c6f3334c43e95f6486942256f77ed2aa926c4d1b/include/mesos/mesos.proto#L3222).

Reply:
this should work as is already. if seccompProfileName is defined, seccompUnconfined is defaulted to false so if seccompProfileName is passed we will always be setting unconfined to false, which is why Unconfined is not an optional and has no if condition above it in PodInfoBuilder